DigitalLearningSolutions.Web/Views/SetPassword/Index.cshtml

```diff
@@ -0,0 +1,50 @@
+@using DigitalLearningSolutions.Web.ViewModels.Common
+@model ConfirmPasswordViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Set Password" : "Set Password";
+}
+
+@section NavMenuItems {
+  <partial name="_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form method="post" asp-action="Index">
+
+      @if (errorHasOccurred)
+      {
+        <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Password), nameof(Model.ConfirmPassword) })" />
+      }
+
+      <div class="nhsuk-u-margin-bottom-8">
+        <h1 id="page-heading" class="nhsuk-heading-xl">Set password</h1>
+        <p class="nhsuk-body-l">
+          Use the form below to set the password for delegate and admin accounts associated with your email address.

```

Comment:
This isn't in scope for this review so doesn't need to be addressed as part of it, but are we making a system where someone (admin or delegate) making a new delegate account at a new centre is asked to reset their established password? That will be odd UX.

Reply:
This functionality is used when a new delegate is created by admins, but no password is set for the account. So delegate won't need to register as the centre does it for them, but will only need to set pw. Hope this makes sense.